src/test/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToUnicodeFormatterTest.java

```diff
@@ -0,0 +1,59 @@
+package net.sf.jabref.logic.formatter.bibtexfields;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class HtmlToUnicodeFormatterTest {
+
+    private final HtmlToUnicodeFormatter formatter = new HtmlToUnicodeFormatter();
+
+    @Before
+    public void setUp() throws Exception {
+        Globals.prefs = JabRefPreferences.getInstance();
```

Comment:
It is best practice to init the class under test in the `@Before` method so that every test method gets a fresh class, i.e move the `= new HtmlToUnicodeFormatter` part to this method.